libjob_queue/src/lsf_driver.c

```diff
@@ -228,6 +228,51 @@ int lsf_job_parse_bsub_stdout(const char * bsub_cmd, const char * stdout_file) {
   return jobid;
 }
 
+/**
+ * Assumes fname points to a file with content "hname1:hname2:hname3" as written by lsf_job_write_bjobs_to_file
+ */
+stringlist_type * lsf_job_parse_bjobs_file(const char* fname) {
```

Comment:
Thank you.  I have addressed all these.  We now never return NULL and the function is called `lsf_job_alloc_parse_hostnames`.